node-tests/plugin-integration/ember-deploy-redis-test.js

```diff
@@ -0,0 +1,40 @@
+/* jshint expr:true */
```

Comment:
quite a bit of cleanup to be done here, but here is the gist thus far and it works locally at the moment.